tools/item_conf_to_yaml_converter.py

```diff
@@ -0,0 +1,202 @@
+                    lastline_was_comment = True
+                
+            if line is '':
+                continue
+            if line[0] == '[':  # item
+                lastline_was_comment = False
+                brackets = 0
+                level = 0
+                closing = False
+                for index in range(len(line)):
+                    if line[index] == '[' and not closing:
+                        brackets += 1
+                        level += 1
+                    elif line[index] == ']':
+                        closing = True
+                        brackets -= 1
+                    else:
+                        closing = True
+                        if line[index] not in valid_chars + "'":
+                            logger.error("Problem (1) parsing '{}' invalid character in line {}: {}. Valid characters are: {}".format(filename, linenu, line, valid_chars))
+                            return config
+                if brackets != 0:
+                    logger.error("Problem parsing '{}' unbalanced brackets in line {}: {}".format(filename, linenu, line))
+                    return config
+                name = line.strip("[]")
+                name = strip_quotes(name)
+                if level == 1:
+                    if name not in config:
+                        config[name] = collections.OrderedDict()
+                    item = config[name]
+                    parents = collections.OrderedDict()
+                    parents[level] = item
+                else:
+                    if level - 1 not in parents:
+                        logger.error("Problem parsing '{}' no parent item defined for item in line {}: {}".format(filename, linenu, line))
+                        return config
+                    parent = parents[level - 1]
+                    if name not in parent:
+                        parent[name] = collections.OrderedDict()
+                    item = parent[name]
+                    parents[level] = item
+
+            else:  # attribute
+                lastline_was_comment = False
+                attr, __, value = line.partition('=')
+                if not value:
+                    continue
+                attr = attr.strip()
+                if not set(attr).issubset(valid_set):
+                    logger.info("line: '{}'".format(line))
+                    logger.error("Problem (2) parsing '{}' invalid character in line {}: {}. Valid characters are: {}".format(filename, linenu, attr, valid_chars))
+                    continue
+                if '|' in value:
+                    item[attr] = [strip_quotes(x) for x in value.split('|')]
+                else:
+                    svalue = strip_quotes(value)
+                    try:
+                        ivalue = int(svalue)
+                        item[attr] = ivalue
+                    except:
+                        item[attr] = svalue.replace('\t', ' ')
+
+        return config
+
+

```

Comment:
I could not use the parser from config.py, i had to modify it. Otherwise, for instance, the comments would have been lost.
Only strip_quotes could have been reused.